src/View/View.php

```diff
@@ -648,7 +648,7 @@ public function setLayout($name)
      *
      * @deprecated 3.5.0 Use getLayout()/setLayout() instead.
      * @param string|null $name Layout file name to set. If null returns current name.
-     * @return string|null
+     * @return string|false
```

Comment:
It can still be null (if not yet set), right?

Reply:
No, the property is set to `'default'` during declaration.

Reply:
You are right, and setTemplate($name) strictly speaking does not accept to make it null again.